README.md

```diff
@@ -328,6 +328,21 @@ Take into consideration that the Notification Extension only lives for a period
 
 While setting up the **setAccessGroup** configuration for the first time, the factors data will migrate to use the **kSecAttrAccessGroup** Keychain's attribute. If the migration fails, error logs will be sent instead of throwing an error during initialization, the data will remain available for the main application but may not be available for App Extensions.
 
+**Unshared Factors from App Groups**
+
+By removing the **App Group** from the **setAccessGroup** configuration, new factors will not be shared via the **Keychain** app groups. 
+
+```swift
+let builder = TwilioVerifyBuilder()
+let twilioVerify = try builder.build()
+```
+
+To unshared existing factors created with **App Groups**, uncheck the **App Group** from the **App/App Extension** configuration in Xcode, as follow:
```

Comment:
And this change should be added to `dev` instead of [feature/notificationExtension](https://github.com/twilio/twilio-verify-ios/tree/feature/notificationExtension)

Reply:
Done, Thank you so much 👍